modules/airgridRtdProvider.js

```diff
@@ -131,7 +141,7 @@ export function passAudiencesToBidders(
   const audiences = getMatchedAudiencesFromStorage();
   if (audiences.length > 0) {
     setAudiencesUsingAppNexusAuctionKeywords(audiences);
```

Comment:
Reviewers: we plan to remove this Xandr specific method / handling in another PR once this is live and tested.